README.md

```diff
@@ -4,6 +4,14 @@ the use of the C based [N-API](https://nodejs.org/dist/latest/docs/api/n-api.htm
 provided by Node.js when using C++. It provides a C++ object model
 and exception handling semantics with low overhead.
 
+There are three options for implementing addons: N-API, nan, or direct
+use of internal V8, libuv and Node.js libraries. Unless there is a need for
+direct access to functionality which is not exposed by N-API as outline
+in [addons.md](https://github.com/nodejs/node/blob/master/doc/api/addons.md#c-addons)
+in Node.js core, use N-API. Refer to
+[C/C++ addons with N-API](https://github.com/nodejs/node/blob/master/doc/api/n-api.md)
+for more information on N-API.
+
```

Comment:
What do you think to link the website doc instead?
- https://nodejs.org/dist/latest/docs/api/addons.html for C++ addons
- https://nodejs.org/dist/latest/docs/api/n-api.html for C++ addon with N-API

Reply:
I had wondered about which would be best, happy to switch to website.

Reply:
@NickNaso fixed.